docs/workflow/testing/libraries/testing.md

```diff
@@ -15,6 +15,11 @@ build.cmd/sh -subset libs.tests
 build.cmd/sh -subset libs.tests -test -c Release
 ```
 
+- The following builds all tests for x86 architecture (**for x86 it's mandatory to build clr** even if you want to run libs tests only):
+```
```

Comment:
Maybe we can add a command for both runtimes. Something like:

To run tests using coreclr:
....

To run tests using mono:
....